docs/wip/how-to-rollback.md

```diff
@@ -1,12 +1,115 @@
 
-- How the rollback works with dynamic and static data?
-- Caching? How much time it takes to invalidate caching
+# 📚 Step by step on how to rollback
+This document is the first version of a guide for the rollback process using VTEX Store Framework Jamstack. It also explains the rollback expected behavior.
+We'll start showing how to make a safe deploy — one that is ready to be used for a rollback.
 
+## 👮‍♀️ How to make a safe deploy (Security Measures)
+- Create a new branch from the current master (this is a backup branch):
+```
+git checkout master
+git pull
+git checkout -b backup-branch
+```
+- Merge your PR, this will trigger a new version to go up.
+
+## 🔙 Rolling back
+If you need to rollback after a deploy, you now have a backup branch named `backup-branch`. 
+An example curl is presented below:
+```
+curl --location --request POST 'http://builders.staging-2a.ingress.vtex.io' \
+--header 'Content-Type: application/json' \
+--header 'Authorization: <VTEX IO token>' \
+--data-raw '{
+    "account": "<account>",
+    "action": "rollback",
+     "head_commit": {
+        "id": "<commit ID>"
+    },
+    "repository": {
+        "name": "<repository Name>",
+        "owner": {
+            "login": "vtex-sites"
+        }
+    },
+    "target": {
+        "branch": "backup-branch"
+    }
+}'
+```
+
+- You can get the `VTEX IO token` in the toolbelt, running the command `vtex local token`;
+- The `commit ID` is the hash that identifies the head commit of the branch `backup-branch`;
+- The repository name is the name of your store's GitHub project, for example `storeName.store`.
+
+Wait for the `2xx` reponse.
+
+# Expected Behavior
+When the rollback is performed with a `2xx` status response, it triggers a new build of your store from the branch `backup-branch`.
+Because of that, the time to replace the current store can take about 10 minutes, depending on the size of your store.
+
+All the subsequent production builds of your store will be based on that commit from the `backup-branch`.
+Let's suppose you where fixing a bug, than after finishing it you and you want to go back to the previously `main` version of your store. Than, you need to perform the request:
+```
+curl --location --request POST 'http://builders.staging-2a.ingress.vtex.io' \
+--header 'Content-Type: application/json' \
+--header 'Authorization: <VTEX IO token>' \
+--data-raw '{
+    "account": "<account>",
+    "action": "rollback",
+     "head_commit": {
+        "id": "<commit ID>"
+    },
+    "repository": {
+        "name": "<repository Name>",
+        "owner": {
+            "login": "vtex-sites"
+        }
+    },
+    "target": {
+        "branch": "main"
+    }
+}'
+```
+If your default branch is called `master` instead of `main`, this will perform a production build with the code in the `main` branch.
+It will also allow new builds from the `main` branch everytime a merge happens.
```

Comment:
```suggestion
>⚠️ *Notice that if your default branch is called `master` instead of `main`, the previous request will run a production build with the code in the main branch. It will also allow new builds from the main branch every time a merge happens.*
```